pymeasure/instruments/common_base.py

```diff
@@ -476,7 +476,13 @@ def fget(self,
                                maxsplit=maxsplit,
                                **values_kwargs)
             if check_get_errors:
-                errors = [str(error) for error in self.check_get_errors()]
+                try:
+                    error_list = self.check_get_errors()
+                except Exception as exc:
+                    log.error("Exception raised while getting a property with the command "
+                              f"""'{command_process(get_command)}': '{str(exc)}'.""")
+                    raise
+                errors = [str(error) for error in error_list]
                 if errors:
                     log.error("Error recieved after trying to get a property with the command "
```

Comment:
"rec**ei**ved" (also elswhere I think). Sorry, it doesn't let me do a suggestion you can merge straight away.

Reply:
Thanks

Reply:
Done